### PR TITLE
Make sure FNULL is closed properly in pycbc.libutils.pkg_config_libdirs

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR modifies the way that a `/dev/null` file object is used in `pycbc.libutils.pkg_config_libdirs` to ensure that said file object actually gets closed properly, the current usage doesn't close the file.

Note that this can all be simplified by using `subprocess.DEVNULL` just as soon as python2 support is dropped.

### CODE REVIEW

**Review by josh-willis — APPROVED** [review]

**Comment by ahnitz** [comment]
@duncanmmacleod BTW, feel free to merge if you've been given approval. Unless there is a reason you are holding off.

**Comment by duncanmmacleod** [comment]
> @duncanmmacleod BTW, feel free to merge if you've been given approval. Unless there is a reason you are holding off.

I don't consider myself a PyCBC maintainer, so would normally leave it to one of y'all.

**Pull request merged**
